app/src/main/java/org/fossasia/openevent/app/event/attendees/AttendeesPresenter.java

```diff
@@ -114,25 +113,14 @@ private void listenToModelChanges() {
         attendeeListener.getNotifier()
             .filter(attendeeModelChange -> attendeeModelChange.getAction().equals(BaseModel.Action.UPDATE))
             .map(DatabaseChangeListener.ModelChange::getModel)
-            .subscribe(this::processUpdatedAttendee, Throwable::printStackTrace);
```

Comment:
The presenter is updating the list on receiving the changes. How is it without any reason. View should not have access to data